yarn.lock

```diff
@@ -1,1553 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Is everyone agree to use `npm` by default or not? I have naver had `yarn` installed and `npm` looks like default